lib/driver_if.cpp

```diff
@@ -1889,33 +1865,18 @@ int driver_set_country_code(void *handle, const char *code)
 	return ret;
 }
 
-static int get_nl80211_bw(wifi_channel_width bw)
-{
-	switch (bw) {
-	case WIFI_CHAN_WIDTH_20:
-		return NL80211_CHAN_WIDTH_20;
-	case WIFI_CHAN_WIDTH_40:
-		return NL80211_CHAN_WIDTH_40;
-	case WIFI_CHAN_WIDTH_80:
-		return NL80211_CHAN_WIDTH_80;
-	case WIFI_CHAN_WIDTH_160:
-		return NL80211_CHAN_WIDTH_160;
-	case WIFI_CHAN_WIDTH_80P80:
-		return NL80211_CHAN_WIDTH_80P80;
-	default:
-		hal_printf(MSG_ERROR, "RTT: bw: %d is not supported!", bw);
-		return -1;
-	}
-}
-
 static void event_handler_sock(struct nl_sock *sock)
 {
-	struct nl_cb *cb = nl_socket_get_cb(sock);
+	struct nl_cb *cb = NULL;
+	
+	if(sock !=NULL){
```

Comment:
-Introduce space between if and (
-Introduce space between != and NULL
-Introduce space between ( and {